### ISSUE TITLE
Doesn't support 'inherit' as value

### ISSUE THREAD

**Issue by cedeber** [ben-eb/colormin#3]
Doesn't work with `color: inherit;` which is a valid declaration.

**Comment by ben-eb** [comment]
colormin should just pass through `inherit`.

```
$ cssnano
h1 { color: inherit; }
```

Outputs:

``` css
h1{color:inherit}
```

**Comment by cedeber** [comment]
Sorry, the problem comes from `-webkit-tap-highlight-color`

**Comment by cedeber** [comment]
```
/usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/lazy-result.js:200
            throw error;
                  ^
Error: Unable to parse color from string "inherit"
    at Object.Color (/usr/local/lib/node_modules/cssnano/node_modules/postcss-colormin/node_modules/color/index.js:31:15)
    at Color (/usr/local/lib/node_modules/cssnano/node_modules/postcss-colormin/node_modules/color/index.js:7:41)
    at /usr/local/lib/node_modules/cssnano/node_modules/postcss-colormin/index.js:26:35
    at /usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:117:34
    at /usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:102:26
    at Rule.each (/usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:89:22)
    at Rule.eachInside (/usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:101:21)
    at /usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:105:32
    at Root.each (/usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:89:22)
    at Root.eachInside (/usr/local/lib/node_modules/cssnano/node_modules/postcss/lib/container.js:101:21)
```

**Comment by ben-eb** [comment]
Fixed by [postcss-colormin@1.2.6](https://github.com/ben-eb/postcss-colormin/commit/937d93b1a317c19ca13d27a17bc738c648599a2f). Thanks for the report. :+1:

**Comment by cedeber** [comment]
You're welcome. Thanks for the quick correction.
